exec/cnex/framestack.h

```diff
@@ -5,12 +5,15 @@
 typedef struct tagTFrame {
     struct tagTCell *locals;
     uint32_t frame_size;
+
+    int stack_depth;
 } TFrame;
 
 typedef struct tagTFrameStack {
     int top;
     int capacity;
     int max;
+    int stack_depth;
```

Comment:
Per the above comments, this shouldn't be here.

Reply:
Removed.